native_client/python/__init__.py

```diff
@@ -95,6 +95,22 @@ def disableExternalScorer(self):
         """
         return deepspeech.impl.DisableExternalScorer(self._impl)
 
+    def addHotWord(self, word, boost):
+        """
+        Add a word and its boost for decoding.
+
+        :param word: the hot-word
+        :type word: str
+
+        :param word: the boost
+        :type word: float
+
+        :throws: RuntimeError on error
+        """
+        status = deepspeech.impl.AddHotWord(self._impl, word, boost)
+        if status != 0:
+            raise RuntimeError("AddHotWord failed with '{}' (0x{:X})".format(deepspeech.impl.ErrorCodeToErrorMessage(status),status))
+
```

Comment:
I guess you also lack `ClearHotWord` and `EraseHotWord` here

Reply:
added in 63ccc